python/jittor/test/test_pytorch_converter.py

```diff
@@ -5,325 +5,260 @@
+    if pretrained:
+        model.load_state_dict(model_zoo.load_url(model_urls['resnet152']))
+    return model
+    
+import numpy as np
+import torch
+import random
+
+# setup random seed
+def setup_seed(seed):
+    np.random.seed(seed)
+    random.seed(seed)
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    torch.backends.cudnn.deterministic = True
+    torch.backends.cudnn.benchmark = True
+"""
 
-        x = torch.Tensor(np.random.rand(16, 3, 15, 15).astype("float32"))
-        with pytorch_converter.trace_scope():
-            y = model(x)
-        ct = pytorch_converter.call_tree
-        ans = """root in:[] out:[]
-    TestModel.forward in:[input_0] out:[out_11] args:{'self': TestModel(
-  (mp1): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-  (mp2): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-  (mp3): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-), 'x': input_0}
-        MaxPool2d.forward in:[input_0] out:[out_1] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': input_0}
-            functional._max_pool2d in:[input_0] out:[out_1]
-        MaxPool2d.forward in:[input_0] out:[out_1] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': input_0} end
-        inj_torch_Tensor___mul__ in:[input_0, input_0] out:[out_5] args:[input_0, input_0]
-        MaxPool2d.forward in:[out_5] out:[out_6] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': out_5}
-            functional._max_pool2d in:[out_5] out:[out_6]
-        MaxPool2d.forward in:[out_5] out:[out_6] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': out_5} end
-        inj_torch_Tensor___add__ in:[out_1, out_6] out:[out_10] args:[out_1, out_6]
-        MaxPool2d.forward in:[out_10] out:[out_11] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': out_10}
-            functional._max_pool2d in:[out_10] out:[out_11]
-        MaxPool2d.forward in:[out_10] out:[out_11] args:{'self': MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False), 'input': out_10} end
-    TestModel.forward in:[input_0] out:[out_11] args:{'self': TestModel(
-  (mp1): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-  (mp2): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-  (mp3): MaxPool2d(kernel_size=3, stride=2, padding=1, dilation=1, ceil_mode=False)
-), 'x': input_0} end
-root in:[] out:[] end"""
-        assert str(ct) == ans
-        code = ct.to_jt()
-        lc = {}
-        exec(code, globals(), lc)
-        print(code)
+@unittest.skipIf(torch is None, "pytorch not found.")
+class TestPytorchConverter(unittest.TestCase):
+    def test_pytorch_converter(self):
```

Comment:
add more model test cases? resnet108 vgg alexnet wideresnet squeezenet etc.

Reply:
No need to test so many models? if only to test converter

Reply:
> add more model test cases? resnet108 vgg alexnet wideresnet squeezenet etc.

No need to test so many models? if only to test converter